pymesos/scheduler.py

```diff
@@ -11,7 +11,8 @@
 class MesosSchedulerDriver(Process, SchedulerDriver):
     _timeout = 10
 
-    def __init__(self, sched, framework, master_uri, use_addict=False):
+    def __init__(self, sched, framework, master_uri,
+                 use_addict=False, implicit_ack=True):
```

Comment:
Just for consistency with documentation of the C++ API (e.g. [this](https://github.com/apache/mesos/blob/master/include/mesos/scheduler.hpp#L391) signature) I might call this `implicit_acknowledgements`. It looks like the Mesos-provided Python bindings don't actually allow the parameter to be passed by keyword, so it's not the end of the world if not.